compiler/src/iree/compiler/Codegen/LLVMGPU/test/vector_distribute.mlir

```diff
@@ -0,0 +1,71 @@
+// RUN: iree-opt --pass-pipeline='builtin.module(hal.executable(hal.executable.variant(builtin.module(func.func(iree-llvmgpu-vector-distribute, canonicalize, cse)))))' -split-input-file %s | FileCheck %s
+
```

Comment:
think Ben added some hook to be able to put the target attribute on the surrounding function or module and not have to write all the hal boiler plate. I think Ben or Hanhan scrubbed or there was an ask to scrub. So lets not add to that

Reply:
Done. The distribution passes need to know the workgroup size and subgroup size, so I added some attribute lookups in the pass to use for testing.